app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt

```diff
@@ -96,7 +97,7 @@ fun WireDialog(
         optionButton2Properties = optionButton2Properties,
         dismissButtonProperties = dismissButtonProperties,
         buttonsHorizontalAlignment = buttonsHorizontalAlignment,
-        modifier = modifier,
+        modifier = modifier.semantics(mergeDescendants = true) {},
```

Comment:
question: what does it do exactly for the dialog? 🤔

Reply:
This is for accessibility. When the dialog is displayed, Talkback will read all it's content not just the title